### PR TITLE
Change serialising of events to use as_json directly

### PR DESCRIPTION
**Pull request by lvonk**
Previously `event.attributes` was used to serialise events. 
Due to this it was not really possible to change the way an object is
serialised to and from JSON.

By introducing `as_json` explicitly for serialising this option becomes
available.

### CODE REVIEW

**Comment by lvonk** [comment]
@bjpbakker @s0meone Please check. Events are no longer serialised via `event.attributes` but via `event.as_json` which adheres more to the `Oj` way of serialising and deserialising. `Oj`s order is:

- `to_hash`
- `as_json`
- `to_json`

By providing `as_json` on `AttributeSupport` which uses `attributes` internally the result should be the same.

**Comment by lvonk** [comment]
Ping @s0meone @bjpbakker

**Comment by lvonk** [comment]
This PR will auto-merge in 1 day... @s0meone @bjpbakker

File: lib/sequent/core/event_store.rb [hunk 1]
```diff
@@ -140,7 +140,16 @@ def store_events(command, streams_with_events = [])
...
           uncommitted_events.each do |event|
-            event_record_class.create!(command_record: command_record, stream_record_id: event_stream.stream_record_id, event: event)
+            values = {command_record: command_record,
+                      stream_record_id: event_stream.stream_record_id,
+                      aggregate_id: event.aggregate_id,
+                      sequence_number: event.sequence_number,
+                      event_type: event.class.name,
+                      event_json: event_record_class.serialize_to_json(event),
+                      created_at: event.created_at}
+            values = values.merge(organization_id: event.organization_id) if event.respond_to?(:organization_id)
```
Comment (bjpbakker) [inline lib/sequent/core/event_store.rb]: Wasn't `organization_id` removed by @derekkraan in an earlier PR?

**Comment by bjpbakker** [comment]
We should probably document that `#to_hash` is used for JSON serialization if both `#to_hash` and `#as_json` are implemented. For me that would be totally unexpected.

**Comment by bjpbakker** [comment]
Other than that, very nice PR! :+1:

File: lib/sequent/core/event_store.rb (hunk 1 above)
Comment (lvonk) [inline lib/sequent/core/event_store.rb]: Yes it was... Let me see how to fix.

**Comment by lvonk** [comment]
Will change organization_id later

**Pull request merged**